<testng-results>
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="6647" started-at="2010-12-24T22:27:02Z" finished-at="2010-12-24T22:27:09Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="6647" started-at="2010-12-24T22:27:02Z" finished-at="2010-12-24T22:27:09Z">
      <class name="com.selenium.test.TestNGDataDrivenTest">
        <test-method status="PASS" signature="setUpServer()" name="setUpServer" is-config="true" duration-ms="6156" started-at="2010-12-24T22:27:02Z" finished-at="2010-12-24T22:27:08Z">
        </test-method>
        <test-method status="FAIL" signature="wikiTest(java.lang.String)" name="wikiTest" duration-ms="0" started-at="2010-12-24T22:27:09Z" finished-at="2010-12-24T22:27:09Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Data Provider public java.lang.String[] com.selenium.test.TestNGDataDrivenTest.dataProvider() must return either Object[][] or Iterator<Object>[], not class [Ljava.lang.String;]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Data Provider public java.lang.String[] com.selenium.test.TestNGDataDrivenTest.dataProvider() must return either Object[][] or Iterator<Object>[], not class [Ljava.lang.String;
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:132)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:391)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1313)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1022)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1122)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1125)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:954)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:883)
	at org.testng.TestNG.run(TestNG.java:817)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="PASS" signature="cleanUp()" name="cleanUp" is-config="true" duration-ms="382" started-at="2010-12-24T22:27:09Z" finished-at="2010-12-24T22:27:09Z">
        </test-method>
      </class>
    </test>
  </suite>
</testng-results>
